setup.cfg

```
@@ -12,3 +12,28 @@ versionfile_source = provenance/_version.py
 versionfile_build = provenance/_version.py
 tag_prefix =
 parentdir_prefix = provenance-
+
+
+[flake8]
+exclude = docs
+ignore = E203,E266,E501,W503,E722,E402,C901,E731,E122
+max-line-length = 100
+max-complexity = 18
+select = B,C,E,F,W,T4,B9
+
+[yapf]
+based_on_style = pep8
```

```suggestion
based_on_style = google
```
I'd prefer to use the google style. I believe it is still pep8 compatible.